src/crypto/transform.py

```diff
@@ -0,0 +1,39 @@
+import json
+from datetime import datetime
+
+from decouple import config
+from google.cloud.storage import Client as BlobClient
+from google.cloud.bigquery import Client as BigQueryClient
+from google.cloud.storage.bucket import Bucket
+from google.cloud.storage.blob import Blob
+
+
+class Transform:
+
+    def __init__(self) -> None:
+        self._blob_client = BlobClient()
+        self._bq_client = BigQueryClient()
+        self._blob = f"{datetime.today().strftime('%Y-%m-%d')}.json"
```

Comment:
saw this in several places, maybe abstract into helper?

Reply:
great suggestion. I started working on making some helper(s) last night to address this. Thanks.